### PR TITLE
add 69. Sqrt

### PR DESCRIPTION
**Pull request by katsurada1**
Please check my code.

### CODE REVIEW

File: 69.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            x (int): A given a non-negative integer.
+        Returns:
+            int: The square root of x.
+        """
+        
+        s = 0
+        r = x
+        while s <= r:
+            mid = (s + r) // 2
+            if mid * mid <= x < (mid + 1) * (mid + 1):
```
Comment (kobakobashu) [inline 69.py:18]: mid ** 2, (mid + 1) ** 2

File: 69.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    def mySqrt(self, x: int) -> int:
+        """A function to compute and return the square root of x.
+        Args:
+            x (int): A given a non-negative integer.
+        Returns:
+            int: The square root of x.
+        """
+        
+        s = 0
+        r = x
```
Comment (kobakobashu) [inline 69.py:15]: what do the s and r mean?

**Review by kobakobashu — CHANGES_REQUESTED** [review]
👀

**Comment by katsurada1** [comment]
I modified the code.
Please check my code.

**Pull request merged**